README.md

```diff
@@ -411,6 +415,8 @@ $transformer = new Transformer(10, function ($data) use ($http) {
 });
 
 $source->pipe($gunzip)->pipe($ndjson)->pipe($transformer)->pipe($dest);
+
+$transformer->on('error', 'printf');
```

Comment:
Should probably be updated to use a proper error handler as well:

```suggestion
$transformer->on('error', function (Exception $e) {
    echo 'Error: ' . $e->getMessage() . PHP_EOL;
});
```

Reply:
May I ask you to check this is used consistently across the examples and documentation?

Reply:
Ok I tried to cover all error statements here. Thanks @clue